modules/apps/27-interchain-accounts/keeper/relay_test.go

```diff
@@ -91,3 +100,128 @@ func (suite *KeeperTestSuite) TestTrySendTx() {
 		})
 	}
 }
+
+func (suite *KeeperTestSuite) TestOnRecvPacket() {
+	var (
+		path       *ibctesting.Path
+		msg        sdk.Msg
+		txBytes    []byte
+		packetData []byte
+		sourcePort string
+	)
+
+	testCases := []struct {
+		msg      string
+		malleate func()
+		expPass  bool
+	}{
+		{
+			"Interchain account successfully executes banktypes.MsgSend", func() {
+				// build MsgSend
+				amount, _ := sdk.ParseCoinsNormalized("100stake")
+				interchainAccountAddr, _ := suite.chainB.GetSimApp().ICAKeeper.GetInterchainAccountAddress(suite.chainB.GetContext(), path.EndpointA.ChannelConfig.PortID)
+				msg = &banktypes.MsgSend{FromAddress: interchainAccountAddr, ToAddress: suite.chainB.SenderAccount.GetAddress().String(), Amount: amount}
```

Comment:
Just a small nit, I think it would increase readability to instantiate on multiple lines like:

```
msg = &banktypes.MsgSend{
    FromAddress: interchainAccountAddr, 
    ToAddress: suite.chainB.SenderAccount.GetAddress().String(), 
    Amount: amount,
}
```

Same below for `data` in other testcases:

```
data := types.IBCAccountPacketData{
    Type: types.EXECUTE_TX,
    Data: txBytes,
}
```

Reply:
Great suggestion. I prefer it as well. I refactored some of the other tests also.